topics/installation.md

```diff
@@ -96,15 +100,14 @@ This way, Valkey will save the data on disk before quitting. Reading the [persis
 
 ## Install Valkey properly
```

Comment:
This feels judgemental. Maybe it would be better to be more descriptive "Install Valkey as a system service"

Also, the section overlooks containers which is how people run it 'properly'.

Reply:
I don't have time to expand it. I just want to delete stuff.

```suggestion
## Install Valkey as a system service
```

Reply:
LGTM